app/views/content/is-teaching-right-for-me/maths/_article.html.erb

```diff
@@ -0,0 +1,92 @@
+<div class="row inset">
+  <section class="col col-720">
+    <p>
+      If maths is your passion, you'll know that it's not just learning formulas or solving equations. It's a way of thinking and problem solving that’s important in every walk of life.
+    </p>
+
+    <p>
+      An exciting career spent passing your knowledge on to the next generation could be for you.
+    </p>
+<p>Tax-free bursaries of £28,000 and scholarships of £30,000 are available for eligible trainee maths teachers.</p>
+  </section>
+
+  <section class="col col-space-s col-content-max">
+    <%= render TextBlockWithTwoImagesComponent.new(
+      heading: "Making maths add up",
+      colour: "pink",
+      image_1_path: "static/images/content/hero-images/0017.jpg",
+      image_1_alt: "",
+      image_2_path: "static/images/content/hero-images/0024.jpg",
+      image_2_alt: "",
+    ) do %>
+      <p>Maths is critical to science, technology and engineering as well as being fundamental for most jobs.</p>
+
+<p>A good understanding of maths provides big benefits for young people and helps create a strong economy.</p>
+
+<p>It's also our government's goal for all young people to study maths to age 18, equipping them with the skills they need for life.</p>
+    <% end %>
+  </section>
+
+  <section class="col col-720 col-horizontal col-space-m">
+    <div>
+      <h2>What you'll be teaching</h2>
+      <p>You'll teach the national curriculum. Themes you’ll cover for 11 to 16 year olds (key stage 3 and 4):</p>
```

Comment:
@caitlin-reidy in the computing page we have a link to the national curriculum at the top and then a link to Oak Academy at the end. It would be good to keep them the same for consistency if there's a suitable maths page to link to.

Reply:
@caitlin-reidy with this and the themes below it would be good, like with computing, to pull out relevant real life examples of the types of things they'll be teaching in the classroom. But we can put a ticket in the backlog to revisit this again later. And also give some examples of the types of jobs people can do with maths (that you may not necessarily think of).